YOZM/YOZM/View/Chapter/StageView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  StageView.swift
+//  YOZM
+//
+//  Created by 정희균 on 9/4/25.
+//
+
+import SwiftUI
+
+struct StageView: View {
+    let stage: Stage
+    let availableWords: [Int64: Bool]
+
+    var body: some View {
+        VStack {
+            ForEach(stage.words.indices, id: \.self) { index in
+                HStack {
+                    if index % 5 == 0 || index % 5 == 4 {
+                        Spacer()
+                    }
+
+                    let newIndex = stage.words.count - index - 1
+                    StageTileView(stage: stage, index: newIndex)
+                        .disabled(
+                            availableWords[stage.words[newIndex].id] == false
+                        )
+
+                    if index % 5 == 2 {
+                        Spacer()
+                    }
```

Comment:
```suggestion
private enum StageLayout {
    static let stageItemCount = 5
    static let spacerPositions = [0, 4]
    static let centerPosition = 2
}
```
여기 이렇게 enum으로 관리하는 거 어떨까요?

Reply:
수정했습니다.